seccomp_test.go

```diff
@@ -590,9 +590,14 @@ func subprocessRuleAddAndLoad(t *testing.T) {
 	}
 	defer filter1.Release()
 
-	call, err := GetSyscallFromName("getpid")
+	const expErr = 28 // ENOSPC, but can be anything not usually returned by listen(2).
```

Comment:
nit: can't we use unix.ENOSPC instead? (or syscall.ENOSPC for that matter, if we are using the deprecated syscall module here, as it seems)

Reply:
You are right, not sure why I overlooked it.

Addressed as a follow up here: https://github.com/seccomp/libseccomp-golang/pull/102